static/js/components/NotificationSettingsSelect.jsx

```diff
@@ -15,7 +15,7 @@ import Tooltip from "@mui/material/Tooltip";
 import grey from "@mui/material/colors/grey";
 import EditNotificationsIcon from "@mui/icons-material/EditNotifications";
 import HelpOutlineOutlinedIcon from "@mui/icons-material/HelpOutlineOutlined";
-import { Box, Slider, Checkbox, Button } from "@mui/material";
+import { Box, Button, Checkbox, Slider } from "@mui/material";
```

Comment:
Same comment as before, please split in 4 distinct imports for webpack splitting

Reply:
✅ Done.